src/dioptra/restapi/experiment/controller.py

```diff
@@ -55,13 +55,35 @@ def __init__(self, *args, experiment_service: ExperimentService, **kwargs) -> No
 
     @login_required
     @responds(schema=ExperimentSchema(many=True), api=api)
-    def get(self) -> List[Experiment]:
-        """Gets a list of all registered experiments."""
+    def get(self) -> Response:
+        """Gets a page or list of all registered experiments."""
         log: BoundLogger = LOGGER.new(
             request_id=str(uuid.uuid4()), resource="experiment", request_type="GET"
         )  # noqa: F841
         log.info("Request received")
-        return self._experiment_service.get_all(log=log)
+
+        index = request.args.get("index", -1, type=int)
```

Comment:
There should be a schema for the parameters accepted in `experiments/schema`, and the `@accepts` decorator used with that schema so that the swagger docs are correctly generated.